lib/client.js

```diff
@@ -831,6 +844,16 @@ class Parser {
   }
 
   destroy () {
+    if (this.ptr) {
+      llhttp.llhttp_free(this.ptr)
```

Comment:
Maybe  we were not freeing memory before?

Reply:
Wasn't needed since the wasm was thrown away.